controllers/operands/kubevirt.go

```diff
@@ -246,13 +250,15 @@ func (*kubevirtHooks) updateCr(req *common.HcoRequest, Client client.Client, exi
 		return false, false, errors.New("can't convert to KubeVirt")
 	}
 	if !reflect.DeepEqual(found.Spec, virt.Spec) ||
-		!reflect.DeepEqual(found.Labels, virt.Labels) {
+		!reflect.DeepEqual(found.Labels, virt.Labels) ||
+		!reflect.DeepEqual(found.Annotations, virt.Annotations) {
```

Comment:
We need to be careful here. Updating the annotations as it done now, will cause `outOfBandUpdate` Alert, each time the user will annotate the KV CR. We need to somehow perform the annotation change, but to not return true in the 2nd return value in line 267, if this is the only change. It's tricky, I know...